zap/src/test/java/org/apache/commons/httpclient/HttpMethodBaseUnitTest.java

```diff
@@ -20,41 +20,29 @@
 package org.apache.commons.httpclient;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
 
 import java.util.List;
-import org.junit.jupiter.api.Test;
+import java.util.stream.*;
+import org.junit.jupiter.params.*;
+import org.junit.jupiter.params.provider.*;
```

Comment:
Please use specific imports, not wildcard.

Reply:
Ok. Uploading a review. Thank you.